<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="mode_frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Programming Modes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_echo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Echo Separation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_machine_learning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine Learning Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="files_frame_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>411</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_file_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>401</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_start_echo_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Echo Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="ai_frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>670</y>
     <width>411</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_file">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>401</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_start_echo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Echo Mode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
